### DISCUSSION TITLE
Linux

### DISCUSSION THREAD

**Discussion by serj-maks** [Homebrew/discussions#5121] [Linux] [answered]
### Additional Info

- [ ] I am using Windows Subsystem for Linux

### Output of `brew config`

```shell
HOMEBREW_VERSION: 4.2.7
ORIGIN: https://github.com/Homebrew/brew
HEAD: 6170b6297c61ef0b68814c708eeff2199d8b0d11
Last commit: 6 days ago
Core tap JSON: 11 Feb 09:55 UTC
HOMEBREW_PREFIX: /home/linuxbrew/.linuxbrew
HOMEBREW_CASK_OPTS: []
HOMEBREW_DISPLAY: :0
HOMEBREW_EDITOR: /usr/bin/nano
HOMEBREW_MAKE_JOBS: 16
Homebrew Ruby: 3.1.4 => /home/linuxbrew/.linuxbrew/Homebrew/Library/Homebrew/vendor/portable-ruby/3.1.4/bin/ruby
CPU: 16-core 64-bit zen3
Clang: 16.0.6
Git: 2.43.0 => /bin/git
Curl: 8.5.0 => /bin/curl
Kernel: Linux 6.6.10-1-MANJARO x86_64 GNU/Linux
OS: Manjaro Linux (Vulcan)
Host glibc: 2.38
/usr/bin/gcc: 13.2.1
/usr/bin/ruby: N/A
glibc: N/A
gcc@11: N/A
gcc: N/A
xorg: N/A
```

### Output of `brew doctor`

```shell
Your system is ready to brew.
```

### Description of issue

hi everyone ! want to install double commander from brew. When typing "double commander" on https://brew.sh/, it switch me to "https://formulae.brew.sh/cask/double-commander#default" page. Here is one double commander default package, usign cask, but linux doesn't have it. If I try to do it, error: "Error: Invalid `--cask` usage: Casks do not work on Linux". Can I install this package or anyone else, if brew suggest cask package ?
P.S. trying to install witout `--cask` keyword has error: 
```
Warning: No available formula with the name "double-commander". Did you mean double-conversion?
==> Searching for similarly named formulae...
==> Formulae
double-conversion

To install double-conversion, run:
  brew install double-conversion
```

**Comment by ZhongRuoyu** [comment]
Casks are not currently supported on Linux. You may want to install the software using one of the installers provided on their website.

**Reply by serj-maks** [reply]
yas, I know that. If brew suggest me install cask double-commander package, this is only way to install it, without alternatives ? 
example
1. I need to install double commander
2. brew suggest me do it with cask
3. linux doesn't support it
4. in this case (I need to install double commander on linux) I need to install it from anywhere else, expext homebrew, case brew has this package (double commander) only in cask format

**Reply by SMillerDev** [reply, marked as answer]
Brew doesn't suggest you install it as a cask. It simply says that's the only version brew has.

Since you're on Linux that doesn't work, so you'll have to find a non-brew way to install it.

**Reply by serj-maks** [reply]
that exactly what I wanted to hear about brew for linux, thanks
